data-serving/samples/line-list.json

```diff
@@ -4,7 +4,7 @@
     },
     "demographics": {
         "age": {
-            "years": 55,
+            "years": 55.0,
```

Comment:
Needed?

Reply:
if importing into mongodb, it assumes it's an int and fails validation unless I REALLY encourage it to be a double